openedx/features/announcements/settings/common.py

```diff
@@ -4,6 +4,17 @@
 def plugin_settings(settings):
     """
     Common settings for Announcements
+    .. toggle_name: FEATURES['ENABLE_ANNOUNCEMENTS']
+    .. toggle_implementation: SettingDictToggle
+    .. toggle_default: False
+    .. toggle_description: This feature can be enabled to show system wide announcements
+    on the sidebar of the learner dashboard. Announcements can be created by Global Staff
```

Comment:
Quick Q: is this indentation correct? Other examples I can find are more indented, like this:

https://github.com/edx/edx-platform/blob/789af8d4aca78175674c5e9b116160dbbf626d4b/lms/djangoapps/courseware/toggles.py#L75-L77

https://github.com/edx/edx-platform/blob/e505d992371dd2dc720134c46ffdf0cd6fe4a2d0/lms/djangoapps/grades/config/waffle.py#L44-L45

https://github.com/edx/edx-platform/blob/b6b6489b9640a967b941de9b1c459669f5a7b952/lms/djangoapps/course_api/__init__.py#L11-L12

Reply:
@bradenmacdonald I have changed the indentation to make it similar to docstring toggle definitions like this
https://github.com/edx/edx-platform/blob/c74b623948b705c71d478235798a0a8896729067/openedx/features/course_duration_limits/models.py#L25-L27

Reply:
Thx! I'll merge now.